src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -5637,8 +5681,19 @@ instruct countTrailingZerosI(rRegI dst, rRegI src, rFlagsReg cr) %{
   ins_pipe(ialu_reg);
 %}
 
+instruct countTrailingZerosI_nf(rRegI dst, rRegI src) %{
+  predicate(UseAPX && UseCountTrailingZerosInstruction);
+  match(Set dst (CountTrailingZerosI src));
+
```

Comment:
Remove this blank line from this and all other patterns.Remove this blank line from this and all other patterns.